packages/react/src/Checkbox/Checkbox.tsx

```diff
@@ -12,7 +12,7 @@ export type CheckboxProps = {
   invalid?: boolean
   /** Allows being neither checked nor unchecked. */
   indeterminate?: boolean
-} & PropsWithChildren<InputHTMLAttributes<HTMLInputElement>>
+} & PropsWithChildren<Omit<InputHTMLAttributes<HTMLInputElement>, 'aria-invalid' | 'type'>>
```

Comment:
> Does omitting attributes from types constitute a breaking change?

I would say so, yes. It is a change in the component’s API, even if they are HTML attributes. We no longer honour any values implementers set and even flag them as type errors, so they must now remove these attributes if they were using them.